.circleci/config.yml

```diff
@@ -133,6 +133,88 @@ jobs:
             - store_test_results:
                 path: test/functional/reports
 
+    functional-tests-without-DRM:
+        executor: dashjs-executor
+        steps:
+            - functional_precondition
+            - checkout
+            - dependencies_setup
+            - build_unit_test_steps
+            - functional_test_setup
+            - run:
+                name: Run functional tests (edge / https) VOD (Static MPD)
+                command:
+                    node test/functional/runTests.js --selenium=remote --reporters=junit --app=remote --browsers=edge --protocol=https --streams="VOD (Static MPD)"
+            - run:
+                name: Run functional tests (edge / https) LIVE (Dynamic MPD)
+                when: always # run tests even if some previous tests failed
+                command:
+                    node test/functional/runTests.js --selenium=remote --reporters=junit --app=remote --browsers=edge --protocol=https --streams="LIVE (Dynamic MPD)"
```

Comment:
Is it possible to define a variable for the browser? Instead of writing `edge` or `chrome` multiple times define that globally?

Reply:
Yes, jobs can be made configurable 
https://circleci.com/docs/2.0/reusing-config/

i suggest defining ONE generic functional test job and then run it with params like:
```
     - functional-test
             browsers: "edge"
             protocol: "https"
             streams: "Live Low Latency"
     - functional-test
             browsers: "chrome"
             protocol: "https"
             streams: "Live Low Latency"
```